lib/landrush/action/redirect_dns.rb

```diff
@@ -24,7 +24,7 @@ def redirect_dns
       def _target_host
         case provider
         when :virtualbox then
-          '10.0.2.2'
```

Comment:
Hmm, what do we really want here? The IP of the gateway to the host or the IP of the nameserver? Is it not the latter? And if so, is _10.0.2.2_ not the correct IP then?